src/mobility/use-is-eligible-for-benefit.tsx

```diff
@@ -1,5 +1,11 @@
 import {useUserBenefitsQuery} from '@atb/mobility/queries/use-user-benefits-query';
 import {OperatorBenefitType} from '@atb-as/config-specs/lib/mobility-operators';
+import {isDefined} from '@atb/utils/presence';
+
+const benefitIdsRequiringValueCode: OperatorBenefitType['id'][] = [
+  'single-unlock',
+  'free-unlock',
+];
 
 export const useIsEligibleForBenefit = (
```

Comment:
Could be a good idea to add tests for `useIsEligibleForBenefit`.
@gorandalum do we have any developer guidelines for considering whether it is worth the time or not to add such tests?

Reply:
Doesn't really have any guidelines on this for now, but when in doubt just add them imo :)

Reply:
Okay, I'll take a look at it before merging then :+1: